pilot/pkg/config/kube/ingressv1/controller.go

```diff
@@ -254,8 +254,10 @@ func (c *controller) SetWatchErrorHandler(handler func(r *cache.Reflector, err e
 	if err := c.ingressInformer.SetWatchErrorHandler(handler); err != nil {
 		errs = multierror.Append(err, errs)
 	}
-	if err := c.classes.Informer().SetWatchErrorHandler(handler); err != nil {
-		errs = multierror.Append(err, errs)
```

Comment:
It was causing nil pointer dereference when `--enableIngressClassName=false` was passed.